File: Kattis/Python/IntroProblems/isithalloween.py
""" 
kattis isithalloween problem
Author: Agis Daniels
Solve simply check if its halloween or xmax
NOTE 
"""

from sys import stdin as rf
def main():
    monSet=set(['OCT 31', 'DEC 25'])
    dat=rf.readline().strip()
    if dat in monSet:
        print("yup")
    else:
        print("nope")
if __name__=='__main__':
    main()

File: Kattis/CPP/IntroProblems/isithalloween.cc
/*
kattis isithalloween problem
Author: Agis Daniels
Solve simply check if its halloween or xmax
NOTE 
*/

#include <bits/stdc++.h>

using namespace std;

int main(){
    string mon;
    int day;
    cin>>mon>>day;
    if(mon=="OCT" && day==31 || mon=="DEC" && day==25){
        cout<<"yup"<<endl;
    }else{
        cout<<"nope"<<endl;
    }
    return 0;
}
